Makefile

```
@@ -189,8 +189,227 @@ bedrock-markdown-links:
+rsk-regtest-restart-log: rsk-regtest-restart take-5 rsk-regtest-log
+
+.PHONY: rsk-regtest-restart-log
+
+# https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#build-the-optimism-monorepo
+rsk-build:
+	pnpm i &&  make op-node op-batcher op-proposer && pnpm build --skipNxCache
+.PHONY: rsk-build
+
+# https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#fill-out-environment-variables
+rsk-env:
+	cp .envrc.example .envrc && direnv allow
+	# sed L1_RPC_URL, L1_CHAIN_ID, L1_RPC_KIND
+	# sed PATH_*s.
+	# Prob also PRIVATE_KEY
+	# and admin, batcher, proposer and sequencer details
+.PHONY: rsk-env
+
+# https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#generate-addresses
+rsk-wallets:
+	direnv allow && \
+	echo "Send 50 rbtc to create2 deployer" && \
+	cast send --from 0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826 --rpc-url "$(L1_RPC_URL)" --unlocked --value 50ether 0x3fAB184622Dc19b6109349B94811493BF2a45362 --legacy && \
+	echo "Send 50 rbtc to GS_ADMIN_ADDRESS" && \
+	cast send --from 0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826 --rpc-url "$(L1_RPC_URL)" --unlocked --value 50ether "$(GS_ADMIN_ADDRESS)" --legacy && \
+	echo "Send 50 rbtc to GS_BATCHER_ADDRESS" && \
+	cast send --from 0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826 --rpc-url "$(L1_RPC_URL)" --unlocked --value 50ether "$(GS_BATCHER_ADDRESS)" --legacy && \
+	echo "Send 50 rbtc to GS_SEQUENCER_ADDRESS" && \
+	cast send --from 0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826 --rpc-url "$(L1_RPC_URL)" --unlocked --value 50ether "$(GS_SEQUENCER_ADDRESS)" --legacy && \
+	echo "Send 50 rbtc to GS_PROPOSER_ADDRESS" && \
+	cast send --from 0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826 --rpc-url "$(L1_RPC_URL)" --unlocked --value 50ether "$(GS_PROPOSER_ADDRESS)" --legacy
+.PHONY: rsk-wallets
+
+# https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#configure-your-network
+rsk-config:
+	[ $(PWD) != $(PATH_CONTRACTS) ] && cd $(PATH_CONTRACTS) && \
+	./scripts/regtest/config.sh
+.PHONY: rsk-config
+
+# add create2 clone, deploy, etc ...
+rsk-create2:
+	[ $(PWD) != $(PATH_CONTRACTS) ] && cd $(PATH_CONTRACTS) && \
+	direnv allow && \
+	cast publish --rpc-url "$(L1_RPC_URL)" 0xf8a58085174876e800830186a08080b853604580600e600039806000f350fe7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf31ba02222222222222222222222222222222222222222222222222222222222222222a02222222222222222222222222222222222222222222222222222222222222222
+.PHONY: rsk-create2
+
+# https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#deploy-the-l1-contracts
+rsk-deploy:
+	[ $(PWD) != $(PATH_CONTRACTS) ] && cd $(PATH_CONTRACTS) && \
+	direnv allow && \
+	forge script scripts/Deploy.s.sol:Deploy -vvv --legacy --slow --sender "$(GS_ADMIN_ADDRESS)" --rpc-url "$(L1_RPC_URL)" --broadcast --private-key "$(GS_ADMIN_PRIVATE_KEY)" --with-gas-price 100000000000
```

I'd use the current gasPrice in testnet/mainnet, remove the sender and use PRIVATE_KEY
```suggestion
	forge script scripts/Deploy.s.sol:Deploy -vvv --legacy --slow --rpc-url "$(L1_RPC_URL)" --broadcast --private-key "$(PRIVATE_KEY)" --with-gas-price 65164000
```

---

done except the private key.